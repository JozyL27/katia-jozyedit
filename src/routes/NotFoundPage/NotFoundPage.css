.NotFoundPage {
  width: 80%;
  margin: 3rem auto;
}

.NotFoundPage > * {
  margin: 1rem 0;
}

.NotFoundPage > h2 {
  font-size: 2.75rem;
  margin-bottom: 2rem;
}

.NotFoundPage > h4 {
  font-size: 1.6rem;
}

.NotFoundPage > p {
  font-size: 1.6rem;
}