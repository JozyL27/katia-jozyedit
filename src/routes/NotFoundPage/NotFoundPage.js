import React, { Component } from 'react'
import './NotFoundPage.css'

export default class NotFoundPage extends Component {
  render() {
    return (
      <section className='NotFoundPage'>
        <h2>404 - Page not found</h2>
        <h4>Fortunately, unlike a corrupted save file, you can just go back to where you last were!</h4>
        <p>Try going back to the previous page.</p>
      </section>
    )
  }
}
