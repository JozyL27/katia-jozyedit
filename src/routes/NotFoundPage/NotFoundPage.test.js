import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'
import NotFoundPage from './NotFoundPage'

describe('<NotFoundPage />', () => {
  // Smoke test
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(
      <MemoryRouter>
        <NotFoundPage />
      </MemoryRouter>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
  // Snapshot test
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <NotFoundPage />
        </MemoryRouter>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})