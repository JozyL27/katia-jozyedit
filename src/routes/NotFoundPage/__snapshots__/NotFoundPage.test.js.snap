// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> renders the UI as expected 1`] = `
<section
  className="NotFoundPage"
>
  <h2>
    404 - Page not found
  </h2>
  <h4>
    Fortunately, unlike a corrupted save file, you can just go back to where you last were!
  </h4>
  <p>
    Try going back to the previous page.
  </p>
</section>
`;
